src/components/action-button.js

```diff
@@ -13,7 +13,11 @@ export const renderButton = ({ layout, theme, app, constraints, senseNavigation,
   const isSense = !!senseNavigation;
   const button = element.firstElementChild;
   const { style, qStateName } = layout;
-  const disabled = layout.useEnabledCondition && layout.enabledCondition === 0;
+  const { navigation } = layout;
+  const { odagLink } = layout;
+  let disabled = layout.useEnabledCondition && layout.enabledCondition === 0;
+  const isOdagLink = navigation.action === 'odagLink' && odagLink.length === 0;
+  disabled = isOdagLink || disabled;
```

Comment:
It will have odagLink as empty string and when the user clicks on that, it sends empty value to the Odag backend. This is resulting in errors. That's why I am disabling it.

Reply:
You also do a check on odagLink in the navigation call itself. That should be enough. Now you are also blocking other actions from happening.

Reply:
Ok, now the button is not disabled, instead the navigation action is restricted based on odagLink value.